src/backend/distributed/metadata/node_metadata.c

```diff
@@ -910,11 +1238,17 @@ ActivateNode(char *nodeName, int nodePort)
 						BoolGetDatum(true));
 	}
 
-	SetUpDistributedTableDependencies(workerNode);
+	SetUpDistributedTableWithDependencies(workerNode);
 
 	if (syncMetadata)
 	{
 		StartMetadataSyncToNode(nodeName, nodePort);
```

Comment:
Well, we should rename `StartMetadataSyncToNode` to something else like `SyncNodeMetadataToNode` or someething?